src/services/Content/ContentQueries.ts

```diff
@@ -37,24 +57,29 @@ export const useContentListQuery = (
     {
       keepPreviousData: true,
       staleTime: 20000,
-      optimisticResults: true,
+      //   optimisticResults: true,
     },
   );
 
 export const useAddContentQuery = (queryClient: QueryClient, request: CreateContentRequest) => {
   const { notify } = useNotification();
   return useMutation(() => AddContentListItems(request), {
-    onSuccess: () => {
-      const title =
-        request?.length > 1
-          ? `${request?.length} custom repositories added`
-          : `Custom repository "${request?.[0]?.name}" added`;
+    onSuccess: (data: CreateContentRequestResponse) => {
+      const hasPending = (data as ContentItem[]).some(({ status }) => status === 'Pending');
+
       notify({
         variant: AlertVariant.success,
-        title,
-        description: 'Repository introspection may take some time',
+        title:
+          request?.length > 1
+            ? `${request?.length} custom repositories added`
+            : `Custom repository "${request?.[0]?.name}" added`,
+        description: hasPending
+          ? 'Repository introspection in progress'
+          : 'Repository introspection data already available',
```

Comment:
This has its own issue, right? Should it be separated from this PR?

Reply:
Good call out, it could be it's own ticket.

I usually allow myself the freedom to amend small bugs as I come across them in the code though. 

What I likely should have done, was mention that specific ticket in the "Additional changes" section of this PR and add this PR to that ticket in Jira

Reply:
you can also mention multiple issues in a title:  Fixes 275,123: Adds "public repositories" tab 
replacing 123 with whatever the other issue # is